### PR TITLE
Finish mobile version portfolio

### PR DESCRIPTION
**Pull request by Odongo006**
It was exciting and fun to complete this project, below are the tasks that I completed in this project : -

- Finished portfolio mobile version
- Created Finish mobile version portfolio branch
- Created a pull request

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -106,45 +113,205 @@ main {
   padding: 0 1rem;
 }
 
-@media screen and (min-width: 900px) {
+.social-icons svg {
+  width: 1.5rem;
+}
+
```
Comment (BertrandConxy) [inline style.css:119]: - [ ] Please add some `margin-top` and `margin-bottom` to this social media icon container so as to add some space around it and also match the design given 😃 👍 
**Given design**
image

**Your design**
image

- [ ] Also please correct the text color for the button below it. You can check the difference in the comparison above 😃

File: style.css [hunk 2]
```diff
@@ -106,45 +113,205 @@ main {
...
+.card-button:hover {
+  background-color: var(--clr-purple);
+  color: var(--clr-white);
+}
+
+.about-btn:hover {
+  background-color: var(--clr-purple);
+  color: var(--clr-white);
+}
```
Comment (BertrandConxy) [inline style.css:188]: - [ ] Please change the text color of your card buttons to blue in order to match the design given 👍 
**Your design**
image
**Given design**
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Odongo006, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @Odongo006, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**